app/snippets/content_templates/isaacQuizSection.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "isaacQuizSection",
+    "encoding": "markdown",
+    "children": [
+        {
+			"type": "content",
```

Comment:
I think this also needs a 'title', looking at the existing quizzes

Reply:
Not necessarily, if the person adds a title, the key will be created, but there's no need for an empty key here. At least, other snippets don't seem to have it.

Reply:
Done.